pkg/statsd/handler_cloud_lookup.go

```diff
@@ -47,7 +47,7 @@ func (ld *lookupDispatcher) run(ctx context.Context) {
 			if err != context.Canceled && err != context.DeadlineExceeded {
 				// This could be an error caused by context signaling done.
 				// Or something nasty but it is very unlikely.
-				ld.logger.Warnf("Error from limiter: %v", err)
+				ld.logger.Warnf("Error from Limiter: %v", err)
```

Comment:
Should probably revert?

Reply:
Done. Still getting used to IntelliJ's refactoring in Go 😖